pandas/api/extensions/__init__.py

```diff
@@ -1,5 +1,5 @@
 """Public API for extending pandas objects."""
-from pandas._libs.lib import _no_default  # noqa: F401
+from pandas._libs.lib import no_default  # noqa: F401
```

Comment:
we want this available to library authors, but we never want users to use this, right?

Reply:
It should be fine for users. I can imagine people working with the signature of functions that use no_default.